.github/workflows/release.yml

```
@@ -6,20 +6,17 @@ on:
-        run: |
-          git clone https://github.com/actions/checkout .
-          git checkout v4
-        working-directory: /tmp/checkout
+   test:
+     name: Pre-release Tests
+     permissions:
+       contents: read
+     runs-on: ubuntu-latest
 
-      - name: Set up Ruby
+     steps:
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 15: did not find expected '-' indicator

(syntax-check)